### ISSUE TITLE
Broken Link on Slide 30

### ISSUE THREAD

**Issue by robcolburn** [alancutter/web-animations-slides-lca-2014#2]
![](https://cloud.githubusercontent.com/assets/245860/8914110/302f278e-3452-11e5-951f-26ac7257dff4.png)

**Comment by alancutter** [comment]
I'm not sure I follow, https://github.com/web-animations/web-animations-js works fine for me.

**Comment by robcolburn** [comment]
Yeah, the URL is right, just the link -- when you click on it from the slide deck.

**Comment by alancutter** [comment]
The link also works fine for me, what URL are you getting?
